soccer/gameplay/plays/Defend_Goal/basic_defend_goal.py

```diff
@@ -45,6 +51,9 @@ def __init__(self, num_defenders=3, num_wingers=2):
             self.add_subbehavior(wing_defender.WingDefender(),
                                  'winger' + str(i))
 
+    def use_standard_defense(self):
```

Comment:
In the future we should rework this to make it a variable each play initializes instead of doing this weird override. Not applicable to this PR though.